deepmd/pd/cxx_op.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+
+def load_library(module_name: str) -> bool:
+    """Load OP library.
+
+    Parameters
+    ----------
+    module_name : str
+        Name of the module
+
+    Returns
+    -------
+    bool
+        Whether the library is loaded successfully
+    """
+    # NOTE: Paddle do not support loading library from .so file yet.
```

Comment:
I am wondering whether this method will be used in the future? In fact, if no C++ codes are built, there is no need to change `backend` directory.